main/guides/getting-started/Explainer - Deploying a Smart Contact.md

```diff
@@ -0,0 +1,22 @@
+# Deploying a Smart Contact
+- To deploy your Agoric Smart Contract, follow the simple steps outlined in this explainer.
+- As you're going through this explainer it may be helpful to watch this video walkthrough, showing you all the steps and the desired outcome of each.
+- TODO: INSERT VIDEO
+
+## Prerequisities:
+- Deploying a Smart Contract requires the Agoric SDK. For steps on installing the Agoric SDK see the Explainer on [Starting a Local Chain](https://github.com/Agoric/documentation/blob/kbennett2000-patch-1/main/guides/getting-started/Explainer%20-%20How%20to%20Start%20a%20Local%20Chain.md)
+
+## Create a Dapp
+- Use the `agoric init` command to initialize a new project.
+- Example: 
+```sh
+agoric init my-first-dapp
+```
+
+- Create a Smart Contract in your `/contract/src` directory
+
+## Deploying Your Smart Contract
+- Create a contract proposal (TODO: add steps)
```

Comment:
did you mean for this PR to be a DRAFT?
TODO shouldn't get merged, right?

Reply:
I don't know who has visibility to draft PRs, I get no notifications for them. Figured Todo was okay because there's nothing linking to this page yet, and there won't be until the video is done. I'll go ahead and remove it though.